benchmark/benchmarkCograph_rec.cpp

```diff
@@ -0,0 +1,55 @@
+#include <iostream>
+#include <map>
+#include <cassert>
+
+#include <io/G6GraphReader.hpp>
+#include "recognition/CographAlg.hpp"
+#include "coloring/CographVertexColoring.hpp"
+#include "independent_set/CographIndependentSet.hpp"
+#include "max_clique/MaxClique.hpp"
+
+int main() {
+    std::map<Koala::CographRecognition::State, int> classification;
+    std::string types[] = {
+            "UNKNOWN",
+            "COGRAPH",
+            "NOT_COGRAPH"
+    };
+
+    while (true) {
+        std::string line;
+        std::cin >> line;
+        if (!std::cin.good()) {
+            break;
+        }
+
+        NetworKit::Graph G = Koala::G6GraphReader().readline(line);
+
+        auto recognize = Koala::CographRecognition(G);
+        recognize.run();
+
+        if (recognize.GetState() == Koala::CographRecognition::State::COGRAPH) {
+            auto IndependentSet = Koala::CographIndependentSet(*(recognize.cotree), G.numberOfNodes());
+            IndependentSet.run();
+
+            assert(("Wrong independent set", IndependentSet.independet_set_size ==
```

Comment:
IndependentSet.independet_set_size -> there should be a method getIndependentSet() returning STL set (as in other independent set algorithms) + invoking size() on it would give you the answer.